docs/gce-quickstart.md

```diff
@@ -0,0 +1,134 @@
+
+## Prerequisite Assumptions
+
+This document assumes that you have:
+
+1. Provisioned your Ansible Control Node, installed Ansible and JMESpath on it, and downloaded the toolkit.
+1. Setup Google Cloud foundational components such as IAM, networking, GSC storage buckets with the required media staged, and security aspects.
+1. Have either Terraform or the Google Cloud CLI and specifically the **gcloud** utility installed.
+
+If you need additional details to setup any of these prerequisites, refer to Google documentation such as [Google Cloud quickstarts and tutorials](https://cloud.google.com/docs/tutorials) and specifically [Install the gcloud CLI](https://cloud.google.com/sdk/docs/install).
+
+## Deploying using the Google Cloud CLI
+
+### Set Supporting Variables
+
+Before beginning, infrastructure locality and networking aspects must be defined and captured (i.e. into shell environment variables for convenience):
+
+```bash
+PROJECT_ID=PROJECT_ID
+REGION_ID=REGION
+ZONE_ID=ZONE
+NETWORK_ID=NETWORK
+SUBNET_ID=SUBNET
+
+gcloud config set project ${PROJECT_ID}
+```
+
+### Create the GCE Instance and Block Storage Devices
+
+Specify some instance specific characteristics such as the VM shape, name, and OS image:
+
+```bash
+MACHINE_TYPE="c4-standard-4"
+IMAGE_FILE="$(gcloud compute images describe-from-family rhel-8 --project=rhel-cloud --format json | jq -r '.selfLink')"
+VM_NAME=INSTANCE_NAME
+```
+
+> **NOTE:** Some operating systems such as Red Hat Enterprise Linux may have additional licensing costs. See the [Premium images](https://cloud.google.com/compute/disks-image-pricing?hl=en#section-1) section of Google documentation for additional details.
+
+Create the instance (add network tags by appending the `--tags TAG` option as required):
+
+```bash
+gcloud compute instances create ${VM_NAME} \
+  --project=${PROJECT_ID} \
+  --zone=${ZONE_ID} \
+  --machine-type=${MACHINE_TYPE} \
+  --network-interface=network-tier=STANDARD,stack-type=IPV4_ONLY,subnet=${SUBNET_ID} \
+  --create-disk=auto-delete=yes,boot=yes,device-name=${VM_NAME}-boot-disk,image=${IMAGE_FILE},mode=rw,provisioned-iops=3300,provisioned-throughput=290,size=64G,type=hyperdisk-balanced
+```
+
+Capture the IP address of the newly created instance:
+
+```bash
+INSTANCE_IP_ADDR=$(gcloud compute instances describe ${VM_NAME} --zone=${ZONE_ID} --format="value(networkInterfaces[0].networkIP)")
+```
+
+Create and attach the desired cloud block storage devices or "disks". Assuming one 64GB disk for the `/u01` file system, and a 500GB disk for each of the ASM `DATA` and `RECO` disk groups:
+
+```bash
+gcloud compute disks create ${VM_NAME}-disk-1 --size=64 --type=hyperdisk-balanced --provisioned-iops=3300 --provisioned-throughput=290 --zone=${ZONE_ID}
```

Comment:
The provisioning of IOPS and throughput are yet more decisions;  is there a default here we can use?